### ISSUE TITLE
fails when asked for diff sample rate 

### ISSUE THREAD

**Issue by sbma44** [mapbox/geojson-tidy#11]
I have a trace I'd like to simplify -- see here: https://gist.github.com/sbma44/ae42ffff66a8ea957c1a

When I specify a sample rate other than the input (1s), only two points are returned. [Example](https://gist.github.com/sbma44/78a59b62aef247951f76).

This is a problem because the map matching API seems to expect data samples at 5s intervals.

cc @planemad

**Comment by planemad** [comment]
@sbma44 saw your comment that the problem was in the exported trace. Is this still open?

**Comment by sbma44** [comment]
Yeah, left this open because I'm still curious about the behavior. My problem was having multiple LineStrings in the submitted FeatureCollection. `geojson-tidy` produced okay results when `minimumTime` is set to 1 but not when set to 5. 

When the traces are combined, however, the default value (5) works fine. So I guess this is okay to close, though perhaps it would be desirable to throw an error when there's more than one feature?
